library/Fission/Random.hs

```diff
@@ -1,42 +1,32 @@
 -- | Random values
 module Fission.Random
-  ( byteString
-  , text
+  ( alphaNum
+  , alphaNumSymbol
   ) where
 
 import qualified Data.ByteString.Random as BS
-import           Data.Word8
 import qualified RIO.ByteString         as BS
+import qualified Fission.Internal.URL   as URL
+import           Data.Text              as T
+import           Data.Word8
+import           GHC.Natural
```

Comment:
Maybe use `Numeric.Natural` instead of the internal `GHC.Natural` module (just as the ghc module suggest).

Reply:
Or just `RIO`, no?

Reply:
Yeah, it's in `RIO` as well it seems: https://www.stackage.org/haddock/lts-14.4/rio-0.1.12.0/RIO-Prelude-Types.html#t:Natural